library/src/main/java/com/mux/video/upload/api/MuxUpload.kt

```diff
@@ -344,13 +345,23 @@ class MuxUpload private constructor(
       return this
     }
 
+    /**
+     * If requested, the Upload SDK will try to standardize the input file in order to optimize it
+     * for use with Mux Video
+     */
+    @Suppress("unused")
+    fun standardizationRequested(enabled: Boolean, maxResolution: MaximumResolution): Builder {
```

Comment:
Should the parameter name be updated to `requested` here?

Reply:
Will add in a separate PR, so we can get this merged.